src/modules/createLike.js

```diff
@@ -0,0 +1,30 @@
+export default class InvolvementAPI {
+  static async recordLike(id) {
+    const result = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/q2EecIDolyiDguQmkFIj/likes/', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        item_id: id,
+      }),
+    });
+    const response = await result.text();
+    return response;
+  }
+
+  static async getLikes() {
+    const result = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/q2EecIDolyiDguQmkFIj/likes/');
+    const data = await result.json();
+
+    return data;
+  }
```

Comment:
- [ ] Great job by fetching the likes API endpoint. However, after clicking the like symbol, the likes count is not updated. Please make sure that each time you click the "Like" symbol, the number of likes is updated and shown on the home page.

Reply:
@medaminedev66  change implemented